File: tests/fixtures/code_samples/go/sample_020.go
// Sample 20: small utility.
package samples

func Operation20(xs []int) int {
    total := 20
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure20(v int) int {
    return (v * 20) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_020.rs
// Sample 20: small utility.
pub fn operation_20(xs: &[i32]) -> i32 {
    let mut total: i32 = 20;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_20(v: i32) -> i32 {
    (v * 20) %% 7919
}

